lib/openstax/aws/stack.rb

```diff
@@ -338,6 +338,47 @@ def status
       end
     end
 
+    def status_reason
+      begin
+        aws_stack.stack_status_reason
+      rescue Aws::CloudFormation::Errors::ValidationError => ee
+        case ee.message
+        when /Stack.*does not exist/
+          "DOES_NOT_EXIST"
+        else
+          raise
+        end
+      end
```

Comment:
"DOES_NOT_EXIST" makes sense as a stack status when the stack does not exist, but it doesn't make sense to me as a status reason.  

Two choices here:

1. This method isn't used, so you could just delete it.
2. If you want to keep it, I'd let it just raise the error, don't rescue anything